apple/Tests/TestCases/Profile/ProfileTests.swift

```diff
@@ -110,4 +110,33 @@ final class ProfileTests: Test<Profile> {
 			decode: { try Profile(jsonData: $0) }
 		)
 	}()
+
+    func test_check_if_profile_json_contains_legacy_p2p_links_when_p2p_links_are_not_present() {
+        func doTest(_ sut: SUT) {
+            XCTAssertEqual(
+                SUT.checkIfProfileJsonContainsLegacyP2PLinks(contents: sut.jsonData()),
+                false
+            )
+        }
+        SUT.sampleValues.forEach(doTest)
```

Comment:
See https://github.com/radixdlt/sargon/pull/123/files#r1596434869, prefer using `eachSample { sut in }` 

I was my mistake having forgot I wrote that.... we should remove all inner `func doTest`...